docs/spec/bank/keeper.md

```diff
@@ -0,0 +1,22 @@
+# Keeper
+
+## Denom Metadata
+
+The BankKeeper contains a store that stores the metadata of different token denoms.  Denoms are referred to by their name, same as the `denom` field in sdk.Coin.  The different attributes of a denom are stored in the denom metadata store under the key `[denom name]:[attribute name]`.  The default attributes in the store are explained below.  However, this can be extended by the developer or through SoftwareUpgrade proposals.
```

Comment:
I don't think we need to abbreviate denom/denoms in this spec for brevity, unless referring to code snippets.

Reply:
Key formats, for sure, but explaining in english sentences, I think the full pronunciation is much clearer (it's even used once or twice later in this spec).